### PR TITLE
Add Cordova sample app created via CLI

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: Samples/UserList/config.xml [hunk 1]
```diff
@@ -1,108 +1,26 @@
...
+    <allow-intent href="http://*/*" />
+    <allow-intent href="https://*/*" />
+    <allow-intent href="tel:*" />
+    <allow-intent href="sms:*" />
+    <allow-intent href="mailto:*" />
+    <allow-intent href="geo:*" />
+    <platform name="android">
+        <allow-intent href="market:*" />
+    </platform>
+    <platform name="ios">
```
Comment (bhariharan) [inline Samples/UserList/config.xml:22]: Nothing here for `<platform name="windows">`. Is that expected?

File: Samples/UserList/hooks/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+-->
+# Cordova Hooks
```
Comment (bhariharan) [inline Samples/UserList/hooks/README.md]: Looks like we have no hooks for this app. Can we remove this?

File: Samples/UserList/platforms/windows/VERSION [hunk 3]
```diff
@@ -0,0 +1 @@
+4.1.0
```
Comment (bhariharan) [inline Samples/UserList/platforms/windows/VERSION:1]: What version is this?

File: Samples/UserList/platforms/windows/www/cordova_plugins.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    {
+        "file": "plugins/com.salesforce/www/com.salesforce.plugin.oauth.js",
+        "id": "com.salesforce.plugin.oauth"
+    },
+    {
+        "file": "plugins/com.salesforce/www/com.salesforce.plugin.sdkinfo.js",
+        "id": "com.salesforce.plugin.sdkinfo"
+    },
+    {
+        "file": "plugins/com.salesforce/www/com.salesforce.plugin.smartstore.js",
```
Comment (bhariharan) [inline Samples/UserList/platforms/windows/www/cordova_plugins.js:12]: These plugins (`SmartStore` and `SmartSync`) should be removed for `Windows`?

File: Samples/UserList/platforms/windows/www/cordova_plugins.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        "runs": true
+    },
+    {
+        "file": "plugins/com.salesforce/src/windows/com.salesforce.plugin.accountmanagerProxy.js",
+        "id": "com.salesforce.SalesforceAccountManagerProxy",
+        "runs": true
+    },
+    {
+        "file": "plugins/com.salesforce/src/windows/com.salesforce.plugin.smartstoreProxy.js",
+        "id": "com.salesforce.SalesforceSmartStoreProxy",
```
Comment (bhariharan) [inline Samples/UserList/platforms/windows/www/cordova_plugins.js:58]: Same as above for `SmartStore` and `SmartSync`.

File: Samples/UserList/platforms/windows/www/js/forcetk.mobilesdk.js [hunk 6]
```diff
@@ -0,0 +1,761 @@
...
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+(function ($j) {
+
+    // Version this js was shipped with
+    var SALESFORCE_MOBILE_SDK_VERSION = "3.3.0";
```
Comment (bhariharan) [inline Samples/UserList/platforms/windows/www/js/forcetk.mobilesdk.js]: This should be `4.0.0`.

File: Samples/UserList/hooks/README.md (hunk 2 above)
Comment (kchitalia) [inline Samples/UserList/hooks/README.md]: Will remove it, provided nothing breaks :)

File: Samples/UserList/platforms/windows/VERSION (hunk 3 above)
Comment (kchitalia) [inline Samples/UserList/platforms/windows/VERSION:1]: This is the Cordova platform version.

File: Samples/UserList/platforms/windows/www/cordova_plugins.js (hunk 4 above)
Comment (kchitalia) [inline Samples/UserList/platforms/windows/www/cordova_plugins.js:12]: Will remove it.

**Comment by bhariharan** [comment]
We should probably remove the `Android` and `iOS` sources here, since it is a `Windows` sample app.

File: Samples/UserList/platforms/windows/www/js/forcetk.mobilesdk.js (hunk 6 above)
Comment (kchitalia) [inline Samples/UserList/platforms/windows/www/js/forcetk.mobilesdk.js]: Will change this to 4.0.0.

**Comment by kchitalia** [comment]
Makes sense for this app to not have the iOS and Android sources. That being said, it is now possible to build cordova apps for iOS and Android on Windows. But I guess we that not what we are showcasing here.

**Comment by bhariharan** [comment]
We don't want to recommend that path for `Android` and `iOS`. It gets tricky around our post-install steps, especially for `Android`. Also, the version of `Cordova` supported is different for `Android`  (5.0.2) and `iOS` (3.9.2). So this will probably not work very well for those platforms.

File: Samples/UserList/config.xml (hunk 1 above)
Comment (kchitalia) [inline Samples/UserList/config.xml:22]: Yep. This is expected.

**Pull request merged**